code/game/supplyshuttle.dm

```diff
@@ -46,6 +46,34 @@ var/list/mechtoys = list(
 	layer = 4
 	explosion_resistance = 5
 
+/obj/structure/plasticflaps/New()
+	..()
+	anchored=0

```

Comment:
Plastic sheet creates new plastic flaps, so I want those to not be anchored.

However, I want the ones already on the map to be anchored.